src/components/alertModal/Index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
@olajide1234 these directories should be named like components. e.g. `AlertModal`.
![image](https://user-images.githubusercontent.com/8125356/56952132-7448ac00-6b31-11e9-8f37-784522998a9b.png)
I can see this is the norm around the app. The team should work on changing this

Reply:
Updated now. Directory names have been fixed.